src/helpers/help.ts

```diff
@@ -50,7 +54,7 @@ export const invariant = (clause: any, message: string, code: ErrorCode) => {
   }
 };
 
-export const warn = (message: string, code: ErrorCode) => {
+export function warn(message: string, code: ErrorCode) {
```

Comment:
preference for block functions?

Reply:
Everything with `asserts` must use the function keyword. Don’t ask me why, it doesn’t make sense 😂

So I converted `warn` as well just to make sure it’s consistent in this file